samples/interop-customvision-textmsg-uwpapp/Documentation/Develop the Windows C# UWP Application.MD

```diff
@@ -0,0 +1,53 @@
+# Text Messaging & Custom Vision Interop Samples
+## Progress
+
+- [x] [Introduction](../README.md)  
+- [x] [Step 1 - Setup Development Environment](./Setup%20DevVM.MD)   
+- [x] [Step 2 - Setup Azure Resources](./Setup%20Azure%20Resources.MD)
+- [x] [Step 3 - Setup Azure IoT Edge for Linux on Windows](./Setup%20Azure%20IoT%20Edge%20for%20Linux%20on%20Windows.MD)  
+- [x] [Step 4 - Develop and publish the IoT Edge Linux module](./Develop%20and%20publish%20the%20IoT%20edge%20Linux%20module.MD)  
+- [x] [Step 5 - Create Certificates for Authentication](./Create%20Certificates%20for%20Authentication.MD)  
+- [ ] **Step 6 - Develop the Windows C# Console Application**  
+- [ ] [Step 7 - Configuring the IoT Edge Device](./Configuring%20the%20IoT%20Edge%20Device.MD)  
+- [ ] [Step 8 - Run samples](./Run%20samples.MD)  
+- [ ] [Troubleshooting](./Troubleshooting.MD) 
+---
+# Step 6: Develop the Windows C# UWP Application
+This part describes how to develop the Windows console application in the development VM. Build the Windows application as an [application package in Visual Studio](https://docs.microsoft.com/windows/msix/package/packaging-uwp-apps):
+
+1 Open the below Windows application based on the sample you are running - Use Visual Studio running as Admin.
+   - Text Messaging: `Windows-IoT-Samples\interop-customvision-textmsg-uwpapp\textmsg-uwpapp\LeafDeviceUWPApp\LeafDeviceUWPApp.sln`
+   - Custom Vision: `Windows-IoT-Samples\interop-customvision-textmsg-uwpapp\customvision\DeviceCustomVisionUWPApp\DeviceCustomVisionUWPApp.sln`
+
+   > **Note:**
+   > Certificate installation
```

Comment:
did you mean "For certificate installation, make sure...."

Reply:
Yes, changed to one  line.